image_geometry/src/pinhole_camera_model.cpp

```diff
@@ -1,5 +1,5 @@
 #include "image_geometry/pinhole_camera_model.h"
-#include <sensor_msgs/distortion_models.h>
+#include <sensor_msgs/distortion_models.hpp>
```

Comment:
I'm honestly not sure.  If we change it in the ROS1 package, then we'll break any downstream includers of it.  I don't think that is desirable, though it isn't the end of the world.  Also, I don't think having it be a different suffix in the ROS2 version is too confusing, and we won't ever try to be directly merging this file from upstream.  Thus, my preference would be to leave the ROS1 version as .h , and make the ROS2 version .hpp.  Thoughts?